setup.sh

```diff
@@ -16,5 +16,9 @@ pipenv run python manage.py migrate
 
 
 # run our app. Nohup and “&” are used to let the setup script finish
+sudo dnf install crontabs
+sudo systemctl start crond
+python manage.py crontab add
```

Comment:
looks like these 3 lines should be above the comment on line 18

Reply:
fixed